lib/postgres.go

```diff
@@ -260,6 +261,16 @@ type PGMetadataLike struct {
 	IsUnlike        bool       `pg:",use_zero"`
 }
 
+// PGMetadataReact represents ReactMetadata
+type PGMetadataReact struct {
+	tableName struct{} `pg:"pg_metadata_reactions"`
+
+	TransactionHash *BlockHash `pg:",pk,type:bytea"`
+	PostHash        *BlockHash `pg:",type:bytea"`
+	IsRemove        bool       `pg:",use_zero"`
+	EmojiReaction   rune       `pg:",type:integer"`
```

Comment:
Are you sure this is an integer? I just frankly don't know off the top of my head.